backend/pkg/helm/release_test.go

```diff
@@ -27,6 +33,11 @@ func GetDefaultKubeConfigPath(t *testing.T) string {
 
 func GetClient(t *testing.T, clusterName string) clientcmd.ClientConfig {
 	t.Helper()
+
+	if os.Getenv("HEADLAMP_RUN_INTEGRATION_TESTS") != "true" {
+		t.Skip("skipping integration test")
```

Comment:
@illume added check to skip integration tests. The integration tests will run only when `HEADLAMP_RUN_INTEGRATION_TESTS` is set to `true`

Reply:
Awesome. Thanks.